src/uu/more/src/more.rs

```diff
@@ -52,9 +52,17 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
 
     // FixME: fail without panic for now; but `more` should work with no arguments (ie, for piped input)
     if let None | Some("-") = matches.value_of(options::FILE) {
-        println!("{}: incorrect usage", executable!());
+        show_usage_error!("Reading from stdin isn't supported yet.");
```

Comment:
please run rustfmt

Reply:
`rustfmt` has no effect on this line. I'll commit anyway, because it changes a line below.